frontend/src/app/modules/boards/index-page/boards-index-page.component.ts

```diff
@@ -10,16 +10,17 @@ import {NewBoardModalComponent} from "core-app/modules/boards/new-board-modal/ne
 import {BannersService} from "core-app/modules/common/enterprise/banners.service";
 import {LoadingIndicatorService} from "core-app/modules/common/loading-indicator/loading-indicator.service";
 import {AuthorisationService} from "core-app/modules/common/model-auth/model-auth.service";
-import {componentDestroyed} from "ng2-rx-componentdestroyed";
 import {enterpriseDemoUrl, enterpriseEditionUrl} from "core-app/globals/constants.const";
 import {DomSanitizer} from "@angular/platform-browser";
 import {boardTeaserVideoURL} from "core-app/modules/boards/board-constants.const";
+import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
+import {componentDestroyed} from "@w11k/ngx-componentdestroyed";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/85e9c35a9c9361e0038637188a645cba25065d78/frontend/src/app/modules/boards/index-page/boards-index-page.component.ts#L18"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>